src/content/documentation/develop/best-practices-for-collecting-user-data-consents.md

```diff
@@ -94,17 +97,21 @@ When your users arrive at your data and privacy consent dialog, they need to kno
 
 ## Prompt after install
 
-As part of your extension’s onboarding flow, include information about your privacy policy and data collection, and seek any necessary user consents. Any privacy information and settings should be clear and unmissable, separating these details from general information about your extension can help.
+As part of your extension’s onboarding flow, include information about your privacy policy and data collection, and seek any necessary user consents. Any privacy information and settings should be clear and unmissable, separating these details from general information about your extension can help. We recommend using a new tab in the currently focussed window.
 
 For more information on how to implement a post-install page or dialog, see [Best practices for onboarding, upboarding, and offboarding users](/documentation/develop/onboard-upboard-offboard-users/).
 
 As mentioned in the [Add-on policies](/documentation/publish/add-on-policies/), if your extension collects user data in association with features that run in the background, such as ad blocking, you need to make sure the data collection is not activated until you have obtained user consent.
 
+It is worth noting that if an update to your add-on adds data collection, updating users must be consented to the newly collected data. The same holds true if your add-on didn't collect any data in the past, but does so in a new update.
```

Comment:
That's not entirely accurate, for example users don't need to consent to newly collected technical data.

Reply:
That's a good point!